pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -320,5 +286,8 @@ func addMutators(mgr kube_ctrl.Manager, rt core_runtime.Runtime, converter k8s_c
 		Scheme:       mgr.GetScheme(),
 	}
 	mgr.GetWebhookServer().Register("/owner-reference-kuma-io-v1alpha1", &kube_webhook.Admission{Handler: ownerRefMutator})
+
+	defaultMutator := k8s_webhooks.DefaultingWebhookFor(converter)
+	mgr.GetWebhookServer().Register("/default-kuma-io-v1alpha1", defaultMutator)
```

Comment:
Will the name change here cause issues during upgrade?

Reply:
Yeah, you can see errors if you try to create a Mesh resource while the CP upgrade is in progress. I can keep both webhooks `/default-kuma-io-v1alpha1` and `/default-kuma-io-v1alpha1-mesh` for backwards compatibility

Reply:
ugh, in this case, it will be called twice for meshes. Maybe we should stick to the old name or document the behavior that webhook can return errors while the upgrade is in process. @michaelbeaumont what do you think?

Reply:
I think leaving the old name is probably the best option, the actual path is relatively unimportant given the CRD.

Reply:
This was resolved